Makefile

```diff
@@ -77,31 +84,38 @@ test/cluster/e2e:
 
 
 $(local_handler_manifest): deploy/handler.yaml
-	mkdir -p $$(dirname $@)
+	mkdir -p $(dir $@)
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)#" \
 		deploy/handler.yaml > $@
 
 $(local_manager_manifest): deploy/operator.yaml
-	mkdir -p $$(dirname $@)
+	mkdir -p $(dir $@)
 	sed "s#REPLACE_IMAGE#$(LOCAL_REGISTRY)/$(MANAGER_IMAGE_FULL_NAME)#" \
 		deploy/operator.yaml > $@
 
+$(CLUSTER_DIR)/%: kubevirtci.version
+	rm -rf kubevirtci
+	git clone https://github.com/kubevirt/kubevirtci
+	cd kubevirtci && git checkout $$(cat  ../kubevirtci.version)
 
-cluster-up:
-	./cluster/up.sh
+cluster-up: $(CLUSTER_UP)
+	$(CLUSTER_UP)
+	$(SSH) node01 -- sudo yum install -y NetworkManager NetworkManager-ovs
```

Comment:
here you use `-- command`, while in other targets ` 'command'`. Please stick to one.

Reply:
It has to be -- command is how it used elsewhere and just 'command' fails.